app/src/main/java/org/ole/planet/myplanet/ui/sync/SyncActivity.kt

```diff
@@ -112,7 +116,9 @@ abstract class SyncActivity : ProcessUserDataActivity(), SyncListener, CheckVers
         } else if (serverConfigAction == "save") {
```

Comment:
the functionality works here when you click the save button. you need to migrate this functionality to the itemSelect click